### ISSUE TITLE
Make content page pass the lighthouse accessibility test

### ISSUE THREAD

**Issue by martinfrances107** [jsdrupal/drupal-admin-ui#178]
This is just a minor follow up to  #137 

When I run localhost:3000 through Chrome's Lighthouse.

I get two accessibility errors  related to  buttons ... inside TablePagination.
That they  do not have a aria-label or equivalent

From lighthouse you can click on a link to get more information 

https://dequeuniversity.com/rules/axe/2.2/button-name?application=lighthouse

TablePagination has the following properties 

backIconButtonProps
nextIconButtonProps

https://material-ui.com/api/table-pagination/

For a page reader, the pagination at the bottom on content.js can be tailored to what is on display, nothing fancy maybe 

Previous content / Next content

**Comment by martinfrances107** [comment]
This is a todo list.

#201 and #196 -- fixed most of the stuff.. 😄 

Regarding the content page ... Lighthouse's A11y score has moved from the low forties  to 83/100.

To get 100/100 we need to fix two issues....

1) The input element  inside the 'TITLE" textField needs a aria-label.
2) The actions element which is greyed out until something is selected....but it  is too greyed out in that it now fails the contrast ratio test.

I am working on a PR now.

**Comment by martinfrances107** [comment]
In the last PR #263   I fixed only -- half the issues on the todo list.

I opened up  #264 so that we can get input from a graph designer on the best way of fixing todo issue 2.

**Comment by martinfrances107** [comment]
There is a side issue for the outstanding issue.

Thank you for looking at this.
